lib/android-tools-info.ts

```diff
@@ -235,5 +337,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 	private getMaxSupportedVersion(): number {
 		return this.parseAndroidSdkString(_.last(AndroidToolsInfo.SUPPORTED_TARGETS.sort()));
 	}
+
+	private validateAndroidHomeEnvVariable(androidHomeEnvVar: string): IFuture<boolean> {
+		return ((): boolean => {
+			let isAndroidHomeCorrect = true;
+			let expectedDirectoriesInAndroidHome = ["build-tools", "tools", "platform-tools", "extras"];
+			if(!androidHomeEnvVar || !this.$fs.exists(androidHomeEnvVar).wait()) {
+				this.printMessage("The ANDROID_HOME environment variable is not set or it points to a non-existent directory. You will not be able to perform any build-related operations for Android.",
+					"To be able to perform Android build-related operations, set the `ANDROID_HOME` variable to point to the root of your Android SDK installation directory.");
+				isAndroidHomeCorrect = false;
+			} else if(!_.any(expectedDirectoriesInAndroidHome.map(dir => this.$fs.exists(path.join(androidHomeEnvVar, dir)).wait()))) {
+				this.printMessage("The ANDROID_HOME environment variable points to incorrect directory. You will not be able to perform any build-related operations for Android.",
```

Comment:
This message does not reflect the check being made. For example, if I forget to install Android extras, but correctly set ANDROID_HOME, I will still get this message.